src/Helper.js

```diff
@@ -0,0 +1,105 @@
+
+let contributors = [
+  {
+    id: 1,
+    name: "Elijah Barreto",
+    cohort: "Echo Cohort 2022",
+    bio:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    linkedin: "https://www.linkedin.com/in/elijah-barreto/",
+    gitHubb: "https://github.com/ElijahBcodes",
```

Comment:
Looks like github is misspelled. I would recommend going with all lowercase `github`